roles/confluent.variables/vars/main.yml

```diff
@@ -145,6 +138,16 @@ kafka_broker_properties:
     properties:
       enable.fips: 'true'
       security.providers: io.confluent.kafka.security.fips.provider.BcFipsProviderCreator,io.confluent.kafka.security.fips.provider.BcFipsJsseProviderCreator
+      ssl.keymanager.algorithm: PKIX
+      ssl.trustmanager.algorithm: PKIX
+      ssl.keystore.type: BCFKS
+      ssl.truststore.type: BCFKS
+      ssl.truststore.location: "{{kafka_broker_truststore_path}}"
+      ssl.truststore.password: "{{kafka_broker_truststore_storepass}}"
+      ssl.keystore.location: "{{kafka_broker_keystore_path}}"
+      ssl.keystore.password: "{{kafka_broker_keystore_storepass}}"
+      ssl.key.password: "{{kafka_broker_keystore_storepass}}"
+      ssl.enabled.protocol: TLSv1.2
```

Comment:
do you need the ssl.enabled.protocol: TLSv1.2 added to each listener? or is here ok?

are any of these props needed if they are added to the listeners